taskcluster/docker/linux/Dockerfile

```diff
@@ -88,8 +88,9 @@ RUN apt-get update -qq \
 
 RUN pip3 install --upgrade pip
 RUN pip3 install \
-    'taskcluster>=4,<5' \
-    pyyaml
+    pyyaml \
+    taskcluster \
+    zstandard
```

Comment:
Needed in a bunch of things, mainly archiving manipulation for some monitoring work that is coming soon.

Reply:
Do you mind adding an in-file comment for consistency with the `apt-get` command above?

Reply:
Done